app/controllers/course/statistics/answers_controller.rb

```diff
@@ -4,6 +4,24 @@ class Course::Statistics::AnswersController < Course::Statistics::Controller
 
   MAX_ANSWERS_COUNT = 10
 
+  def latest_attempt
```

Comment:
I feel this should be renamed as `latest_answer` to communicate that we are retrieving an answer.

Reply:
Done